hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/meta/queries/GetVersionInfoResourceUsage.java

```diff
@@ -41,4 +41,13 @@ public boolean applicableTo(final Query query) {
     public FeeData usageGiven(final Query query, final StateView view, final Map<String, Object> ignoreCtx) {
         return FixedUsageEstimates.getVersionInfoUsage();
     }
+
+    /**
+     * This method is used to calculate the fee for the {@code NetworkGetVersionInfo}
+     * query in modularized code only.
+     * @return the fee for the query
+     */
+    public static FeeData usageGiven() {
+        return FixedUsageEstimates.getVersionInfoUsage();
+    }
```

Comment:
This `FixedUsageEstimates.getVersionInfoUsage()` is static can we directly call `getVersionInfoUsage()` in the `GetVersionInfoHandler.calculateFees()` ?

Reply:
Done